### ISSUE TITLE
Classes in 'defaults' config not being appended to dom

### ISSUE THREAD

**Issue by anthonycollini** [rwwagner90/ember-shepherd#32]
When I set classes in my 'defaults' config, they get added to the object, but do not translate to the dom. Am I missing something?

As you can see, I added `modal` and `myclass` to the defaults classes, and they do not show up in the dom.

<img width="1171" alt="screenshot 2016-01-15 11 11 04" src="https://cloud.githubusercontent.com/assets/933222/12358657/60702180-bb7c-11e5-89cc-6f207cf96d0f.png">

<img width="1076" alt="screenshot 2016-01-15 11 13 52" src="https://cloud.githubusercontent.com/assets/933222/12358656/606fedc8-bb7c-11e5-998b-c0b99cde7069.png">

Here is my defaults object:

    this.get('tour').set('defaults', {
      classes: 'myclass modal shepherd-element shepherd-open shepherd-theme-arrows',
      modal: true,
      scrollTo: false,
      showCancelLink: true,
      copyStyles: false,
      highlightClass: 'highlight'
    });

**Comment by anthonycollini** [comment]
I am also finding that most of the options do not work, such as disableScroll, scrollTo, and modal, which are the only ones I have been using. I am able to scroll and there is no modal, or styles for that matter.

` "ember": "2.2.0"`
`"ember-shepherd": "1.1.0",`

**Comment by anthonycollini** [comment]
I created an example app that shows the issues, which include the following options not working:

- custom classes
- disableScroll
- modal
- no styles
- you can't run the tour with a `builtInButtons` array.

https://github.com/anthonycollini/ember-shepherd-test

**Comment by rwwagner90** [comment]
@anthonycollini I'm looking into your example, but our sample app uses ember 2.2.0 and ember-shepherd 1.1.0 and everything seems to work for me. Were you using a previous version of ember-shepherd with success?

**Comment by rwwagner90** [comment]
@anthonycollini you are not setting these things correctly. `defaults` is for Shepherd options only, not for ember-shepherd options. Please look at the docs again. You need to do something like this:

```js
    this.get('tour').set('defaults', {
      classes: 'my-class modal',
      scrollTo: false,
      showCancelLink: true
    });

    this.get('tour').set('disableScroll', true);
    this.get('tour').set('modal', true);
```

**Comment by rwwagner90** [comment]
I sent a PR your way to fix your example https://github.com/anthonycollini/ember-shepherd-test/pull/1

**Comment by rwwagner90** [comment]
Please read the ember-shepherd docs and the Shepherd docs, then let me know if anything is still not working.

**Comment by anthonycollini** [comment]
Okay, I see you can't pass those options to the defaults hash. Modals and disableScroll are now working. However, it still seems as the classes are not being passed properly. My 'modals' do not have not have any style, and I can't pass custom classes through defaults, as well as through the individual steps as noted in my initial comment in this thread with screenshot. Any thoughts on that?

**Comment by rwwagner90** [comment]
@anthonycollini I checked out the class issue, and you are right about that. Working on a fix now.

**Comment by rwwagner90** [comment]
@anthonycollini I think I fixed this issue, but it still doesn't behave like I would assume it would. I would assume defaults and the classes you define on a per step basis would sort of merge and all be applied for that step, but I am not sure Shepherd supports that. I posted an issue there to determine: https://github.com/HubSpot/shepherd/issues/90

Also, after this fix I made, you will need to make sure you set defaults **before** setting the steps.
